npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -54,6 +53,25 @@ var (
 	ingressOrEgressPolicyChainPattern = fmt.Sprintf("'Chain %s-\\|Chain %s-'", util.IptablesAzureIngressPolicyChainPrefix, util.IptablesAzureEgressPolicyChainPrefix)
 )
 
+type osTools struct {
+	chainsToCleanup map[string]struct{}
+}
+
+func makeTools() osTools {
+	return osTools{make(map[string]struct{})}
+}
+
+func (pMgr *PolicyManager) reboot() error {
+	// TODO for the sake of UTs, need to have a pMgr config specifying whether or not this reboot happens
```

Comment:
ya we have reset which cleans up and deletes, and initialize which reinstalls. This `reboot` will do nothing in Windows

Reply:
Just curious. 
I guess you decide to do a pro-active approach to avoid time to install default chains when the first `networkpolicy` comes again while logically only `reset` is called when there is no more `networkpolicy` and `initialize` is called if the first networkpolicy comes.

Reply:
oh I see your point. In v1 it looks we just reset, and then when the first policy comes in, we initialize. 

Given that the current DP design initializes the pMgr on creation, it might be simpler to always have pMgr initialized. Need to consider if there are any security or perf concerns for this approach.